src/localization/locales/en-US.json

```diff
@@ -293,7 +293,14 @@
         "list_map_help_url": "https://terraso.org/help/add-a-new-landscape-to-terraso/",
         "list_map_section_label": "Landscapes map",
         "list_map_popup_link": "View details about {{name}}",
-        "boundaries_update_save": "Update Map"
+        "boundaries_update_save": "Update Map",
+        "form_boundary_draw_polygon_title": "[TODO] Draw {{name}}’s Boundary",
+        "form_boundary_draw_polygon_description": "[TODO] Click anywhere to start drawing. Save your drawing and return to complete later. You can adjust the boundary in edit mode. ",
+        "form_boundary_draw_polygon_help": "[TODO] Need help?",
+        "form_boundary_draw_polygon_help_url": "https://terraso.org/help/",
```

Comment:
question: Should this go to drawing-specific help?

Reply:
@paulschreiber Hana asked to put that general link, see the converastion here: https://tech-matters.slack.com/archives/C02K02L3Q48/p1659022864655649

Reply:
yeah, that was "for now" — we will need drawing-specific help